Packs/CommonScripts/Scripts/FormatURL/FormatURL.py

```diff
@@ -34,7 +34,10 @@ class URLCheck(object):
     This class will build and validate a URL based on "URL Living Standard" (https://url.spec.whatwg.org)
     """
     sub_delims = ("!", "$", "&", "'", "(", ")", "*", "+", ",", ";", "=")
-    brackets = ("\"", "'", "[", "]", "{", "}", "(", ")", ",")
+    brackets = ("\"", "'", "[", "]", "{", "}", "(", ")")
+    url_code_points = ("!", "$", "&", "\"", "(", ")", "*", "+", ",", "-", ".", "/", ":", ";", "=", "?", "@",
+                            "_", "~")
```

Comment:
what is the purpose of this change?

Reply:
I removed the comma from the `brackets` as it isnt a bracket and caused issues with the flow of the script. The comma is allowed and it is in the `url_code_points` but not as a bracket.

Reply:
there is a definition of `url_code_points` also in `check_codepoint_validity()`, meaning what you defined will not be used there

Reply:
Oh, wait, what, how did that even get there! I just deleted the comma from brackets and somehow must have pasted it there by mistake.